unsubscribe.html

```diff
@@ -353,72 +360,100 @@ <h5 id="unsubscribe-intro">{% filter ak_text:"unsubscribe_intro" %}{% endfilter
 					<div id="id_email_box" class="input-text ak-input-type-user required ak-err-below">
 						<label for="id_email">{% filter ak_text:"field_email" %}Email Address{% endfilter %}</label>
 						<input type="text" name="email" id="id_email" class="ak-userfield-input">
-					</div>			 
-	<!--		   
+					</div>
+	<!--
 				<div class="ak-labels-overlaid">
 					<div>
 						<label for="id_email">{% filter ak_text:"field_email"|capfirst %}{% endfilter %}</label>
 						<input id="id_email" type="text" name="email" size=40 />
 					</div>
 					<input type="hidden" name="country" value="United States" />
 				</div>
-	-->	
+	-->
 			</div>
 			<div id="known_user">
-			<div>{% filter ak_text:"unsubscribe_user_warning_prename " %}Not{% endfilter %} <strong><span id="known_user_name"></span></strong>{% filter ak_text:"unsubscribe_user_warning_postname " %}?{% endfilter %} <a href="/unsubscribe/unsubscribe">{% filter ak_text:"unsubscribe_user_warning_link_label" %}Use this form instead{% endfilter %}</a></div>				
+				<div>
+					{% filter ak_text:"unsubscribe_user_warning_prename " %}Not{% endfilter %}
+					<strong>
+						<span id="known_user_name"></span>
+					</strong>
+					{% filter ak_text:"unsubscribe_user_warning_postname " %}?{% endfilter %}
+					{% if page.lang.iso_code == "de"%}
+						<a href="/unsubscribe/unsubscribe-de">
+					{% elif page.lang.iso_code == "fr" %}
+						<a href="/unsubscribe/unsubscribe-fr">
+					{% elif page.lang.iso_code == "es" %}
+						<a href="/unsubscribe/unsubscribe-es">
+					{% elif page.lang.iso_code == "pt" %}
+						<a href="/unsubscribe/unsubscribe-pt">
+					{% elif page.lang.iso_code == "jp" %}
+						<a href="/unsubscribe/unsubscribe_japanese">
+					{% elif page.lang.iso_code == "tr" %}
+						<a href="/unsubscribe/unsubscribe-turkish">
+					{% elif page.lang.iso_code == "se" %}
+						<a href="/unsubscribe/unsubscribe-sw">
+					{% elif page.lang.iso_code == "nl" %}
+						<a href="/unsubscribe/unsubscribe-du">
+					{% elif page.lang.iso_code == "pl" %}
+						<a href="/unsubscribe/unsubscribe-pl">
+					{% else %}
+						<a href="/unsubscribe/unsubscribe">
+					{% endif %}

```

Comment:
it seems like this will work appropriately. just noting that it seems there may be a few other language versions to add here, such as chinese, bengali, etc

Reply:
We don't have the unsubscribe pages translated into those languages (just piloting our email lists there), but that's a good flag for when we do, thanks!